public/javascripts/lists.js

```diff
@@ -51,19 +74,19 @@ document.addEventListener('DOMContentLoaded', async (event) => {
                         const dateValue = document.getElementById("task-date-edit").value;
                         const timeValue = document.getElementById("task-time-edit").value;
                         const experienceValue = document.getElementById("task-exp-edit").value;
-
                         const res = await fetch(`/tasks/${elem.id}`, {
                             method: "PUT",
                             headers: {
                                 "Content-Type": "application/json"
                             },
-                            body: {
```

Comment:
We don't want these lines removed because it will break our task editing function